libs/cypress/fixtures/enrollmentRequests/index.ts

```diff
@@ -5,7 +5,7 @@ const approvedErStatus = {
     approved: true,
     approvedAt: '2024-04-10T12:13:45Z',
     approvedBy: 'unknown',
-    labels: { displayName: 'east-device', fleet: 'eu-east-prod-001', region: 'spain' },
+    labels: { 'display-name': 'east-device', fleet: 'eu-east-prod-001', region: 'spain' },
```

Comment:
we dont need to change this. `displayName` is valid label key

Reply:
You are right, I'll revert these changes.